### PR TITLE
Task Contact Info Pull Request

### PR DESCRIPTION
**Pull request by Knux2**
Please review my code

### CODE REVIEW

**Comment by Knux2** [comment]
Changed Merge destination to Story Branch

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+}
+
+.contactSymbols {
+    text-align: center;
+    height: 100px;
+    background-color: #577399;
+}
+
+.contactSymbols ul {
+    list-style: none;
```
Comment (rrrichard) [inline css/style.css]: line 155 display:inline-block gets rid of the list-style automatically.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+}
+
+.contactSymbols li {
+    display: inline-block;
+}
+
+.icon {
+    margin: 15px;
+    height: 70px;
+    width: 70px;
```
Comment (rrrichard) [inline css/style.css:164]: height and width: don't remove yet but the size of the icon is being controlled at some other attributes. could be useful in later edits.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                <div class="projectInfo"></div>
+            </div>
+        </main>
+    </div>
+
+    <div class="contactInfo" id="contactInfo">
+        <h1 class="contactTitle">Contact</h1>
+    </div>
+    <div class="contactSymbols">
+        <ul>
```
Comment (rrrichard) [inline index.html:67]: contactSymbols <ul>'s margin is making the bottom of the contact links have a white space at the bottom. Could be a proble if you are attempting to have no white space at the bottom of screen

Comment (rrrichard) [reply index.html:67]: height: 120px; seems to cover the bottom with blue. There might be a better fix.

**Review by rrrichard — COMMENTED** [review]
Review yourself before taking my comments into consideration:

File: css/style.css (hunk 2 above)
Comment (Knux2) [reply css/style.css:164]: Changing both does effect the size of the icons

File: index.html (hunk 3 above)
Comment (Knux2) [reply index.html:67]: Changed height within CSS

**Review by rrrichard — APPROVED** [review]

**Pull request merged**